include/seqan/align/dp_formula_affine.h

```diff
@@ -112,12 +112,13 @@ _computeScore(DPCell_<TScoreValue, AffineGaps> & current,
                     TTracebackConfig{});
     if (IsLocalAlignment_<TAlgorithm>::VALUE)
     {
+        using TMaxScorePolicy = typename TraceConfigForLocalAlignment_<TTracebackConfig>::Type;
```

Comment:
I hope you found all occurrences :)

So much duplicated code.

Reply:
yes I know. But I did not want to start refactoring this now.